tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -1,99 +1,57 @@
 import { myUser, opponentOne } from '../../fixtures/userFixtures';
 
-function setup(isRanked = false) {
-  cy.wipeDatabase();
-  cy.visit('/');
-  cy.signupPlayer(myUser);
-  cy.createGamePlayer({ gameName: 'Test Game', isRanked }).then((gameSummary) => {
-    cy.window()
-      .its('cuttle.gameStore')
-      .then((store) => store.requestSubscribe(gameSummary.gameId));
-    cy.vueRoute(`/lobby/${gameSummary.gameId}`);
-    cy.wrap(gameSummary).as('gameSummary');
-  });
-}
 
-function checkRanked(ranked) {
-  cy.window()
-  .its('cuttle.gameStore')
-    .then((game) => {
-      expect(game.isRanked).to.eq(ranked);
-    });
-}
+import en from '../../../../src/translations/en.json';
 
-function assertGameStarted(noMovesYet = true) {
-  cy.url().should('include', '/game');
-  cy.window()
-    .its('cuttle.gameStore')
-    .then((game) => {
-      expect(game.players[1].hand.length).to.eq(6);
-      if (noMovesYet) {
-        expect(game.players.length).to.eq(2);
-        expect(game.players[0].hand.length).to.eq(5);
-        expect(game.deck.length).to.eq(39);
-      }
-      expect(game.topCard.rank).to.be.greaterThan(0);
-      expect(game.secondCard.rank).to.be.greaterThan(0);
-      expect(game.scrap.length).to.eq(0);
-      expect(game.twos.length).to.eq(0);
-    });
-}
```

Comment:
I generally think it's better to have the helper functions at the top of the file, this way I can browse through them first before looking at test, especially in files with hundreds if not thousands of lines

Reply:
yeah I see what you mean.  My idea was that with explicit function names, you don't need to check what they do to understand the test.

Reply:
I see the intention you were going for though. A bit of advice that I try and adhere to when working on individual tickets; keep the concerns separated. 

This PR has a specific, targeted fix and while the other changes might be valuable it’s probably better to fork that off to a new issue so we can discuss your reasoning and come up with a better solve to be fixed in a separate update.